mautic/mautic
# Remove TemplatingHelper

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Once this PR is merged the following files must be removed: 
- `app/bundles/CoreBundle/Templating/TemplateNameParser.php`
- `app/bundles/CoreBundle/Templating/TemplateReference.php`
- `templating.name_parser` from config



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Use themes in preference center, page and check the theme loads as expected
